### PR TITLE
added edit feature to team

### PR DESCRIPTION
**Pull request by dasolyoon**

### CODE REVIEW

File: features/overall_wrapper.feature [hunk 1]
```diff
@@ -6,22 +6,17 @@ Feature: A user can edit their profile
...
-  Scenario: An edit button is on the home page
-    Given I am on the home page
-    Then I should see "Edit"
-
   Scenario: Successfully edit my account information
     Given I am on the team_index page
-    And I should see "Edit"
     Then I press "Edit My Info"
-    And I fill in "Name" with "EECSxxSahaxxEECS"
+    And I fill in "Name" with "EECSxxSahaixxEECS"
```
Comment (jjlgao) [inline features/overall_wrapper.feature]: you would think we would be more serious on a professional tool

File: app/controllers/team_controller.rb [hunk 2]
```diff
@@ -35,9 +35,9 @@ def leave
     return redirect_to without_team_path
   end
   
-  # TODO: edit users in the team page (for now, it's just text)
-  # but we need to change that to a button + add some kind of form?
-  # what were we going to do with that anyway, add/delete?
-  # we could just have a "remove" button and some kind of 
-  # "Are you sure?" thing--can ask MDS what he wants idk
+  def edit
+    @user_to_remove = User.find(params[:unwanted_user])
```
Comment (jjlgao) [inline app/controllers/team_controller.rb]: Should we have permissions here to ensure the user is on the correct team, you're on the same team as that user, etc? I don't see that check being made anywhere, and I think we should probably still deal with security issues given that with or without a UI the backend is still RESTful.

File: features/overall_wrapper.feature (hunk 1 above)
Comment (dasolyoon) [inline features/overall_wrapper.feature]: We probably have to replace all our names later tbh, but fixed

File: app/controllers/team_controller.rb (hunk 2 above)
Comment (dasolyoon) [inline app/controllers/team_controller.rb]: Yeah I wrote down a note to myself, but we have to do this later on where we do some before_action events to check who's logged in/has permissions in all of our controllers

File: features/overall_wrapper.feature [hunk 3]
```diff
@@ -16,7 +16,7 @@ Feature: A user can edit their profile
   Scenario: Successfully edit my account information
     Given I am on the team_index page
     Then I press "Edit My Info"
-    And I fill in "Name" with "EECSxxSahaixxEECS"
+    And I fill in "Name" with "GaoxxJasonxxGao"
```
Comment (jjlgao) [inline features/overall_wrapper.feature]: i still disapprove

**Comment by jjlgao** [comment]
Lgtm

**Pull request merged**